UnchainedLauncher.Core/Services/Mods/ModManager.cs

```diff
@@ -50,6 +50,10 @@ public ModManager(
             EnabledModReleases = new ObservableCollection<Release>(enabledMods);
         }
 
+        // simply copy constructor
+        // assumes Release elements are immutable so the shallow copy is ok
+        public ModManager(ModManager other) : this(other.ModMap, new(other.EnabledModReleases)) { }
```

Comment:
I think this should be a method rather than a constructor

```
public ModManager Copy() = new ModManager(this.ModMap, this.EnabledModReleases)
```